src/plugin/protocol-htmlunit/src/java/org/apache/nutch/protocol/htmlunit/HttpResponse.java

```diff
@@ -40,6 +42,7 @@
 import org.apache.nutch.net.protocols.HttpDateFormat;
 import org.apache.nutch.net.protocols.Response;
 import org.apache.nutch.protocol.ProtocolException;
+import org.apache.nutch.protocol.httpclient.DummyX509TrustManager;
```

Comment:
This import is not available, i.e., the code does not compile. Could verify this by running `ant clean runtime`?

Plugins are encapsulated and classes from one plugin are not visible in other plugins except they are explicitly requested in the plugin.xml. However, in this case we just could copy the DummyX509TrustManager into the package `org.apache.nutch.protocol.htmlunit`.